install.sh

```diff
@@ -128,7 +128,10 @@ Run Coder:
   # Or just run the server directly
   $ coder server
 
-Use Coder:
+  Default URL: http://127.0.0.1:3000
```

Comment:
I think the Default URL is `:8080`

Reply:
this is for `./scripts/develop.sh`, but not when you install the binary/package